### PR TITLE
Chore: Notify in slack when nightly docker image scan fails

### PR DESCRIPTION
**Pull request by idafurjes**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -3457,6 +3457,17 @@ steps:
   - trivy --exit-code 1 --severity HIGH,CRITICAL grafana/grafana:latest-ubuntu
   - trivy --exit-code 1 --severity HIGH,CRITICAL grafana/grafana:main-ubuntu
 
+- name: slack
+  image: plugins/slack
+  settings:
+    channel: grafana-backend
+    template: Nightly docker image scan job for {{repo.name}} failed
```
Comment (dsotirakis) [inline .drone.yml:3464]: Maybe we can add some more info - links to drone failed pipeline page?

(check template references [here](http://plugins.drone.io/drone-plugins/drone-slack/))

File: scripts/job.star [hunk 2]
```diff
@@ -50,3 +53,17 @@ def scan_docker_image_high_critical_vulnerabilities_step(edition):
         'image': 'aquasec/trivy:0.18.3',
         'commands': commands,
     }
+
+def slack_job_failed_step(channel):
+    return {
+        'name': 'slack',
```
Comment (dsotirakis) [inline scripts/job.star:59]: A different name - like `notify_failure` should do be also fine?

**Review by dsotirakis — COMMENTED** [review]
Great - 2 niiiiit comments!

File: .drone.yml (hunk 1 above)
Comment (dsotirakis) [reply .drone.yml:3464]: (another source [here](https://github.com/grafana/grafana/blob/main/scripts/lib.star#L76))

File: scripts/job.star (hunk 2 above)
Comment (idafurjes) [reply scripts/job.star:59]: sure

File: scripts/job.star [hunk 3]
```diff
@@ -50,3 +53,17 @@ def scan_docker_image_high_critical_vulnerabilities_step(edition):
         'image': 'aquasec/trivy:0.18.3',
         'commands': commands,
     }
+
+def slack_job_failed_step(channel):
+    return {
+        'name': 'slack_notify_failure',
```
Comment (aknuds1) [inline scripts/job.star:59]: Nit, but better be consistent as we use dashes in Drone names elsewhere :)
```suggestion
        'name': 'slack-notify-failure',
```

**Review by aknuds1 — APPROVED** [review]
LGTM!

**Comment by aknuds1** [comment]
Remember to set the milestone.

**Pull request merged**